client/components/marketing-survey/gsuite-cancel-purchase-dialog/index.jsx

```diff
@@ -207,7 +209,10 @@ class GSuiteCancelPurchaseDialog extends Component {
 					onClose={ onClose }
 				>
 					{ steps.GSUITE_INITIAL_STEP === this.state.step ? (
-						<GSuiteCancellationFeatures purchase={ purchase } />
+						<GSuiteCancellationFeatures
+							googleSubscriptionStatus={ getGSuiteSubscriptionStatus( selectedDomain ) }
```

Comment:
I believe we don't need to retrieve that domain name since it is already available from the `purchase` object. We actually display that domain name in the modal:

> If you cancel your subscription for **example-workspace.com** now, you will lose access to all of your Google Workspace features in 30 days. After that time, you will need to purchase a new subscription with Google.

We could also retrieve that subscription status from inside `GSuiteCancellationFeatures` since that component is connected to the Redux global state tree already.

Reply:
We actually need it @stephanethomas, since the function does not accept a string, but a domain object.